### ISSUE TITLE
Unexpected and undesirable Search functionality looking for "beanproperty"...

### ISSUE THREAD

**Issue by jim-oflaherty-jr-qalocate-com** [scalacenter/scala-3-migration-guide#174]
My first experience arriving at the home page is already undesirable and immediately diminished my migration confidence. It happened as a result of [this thread](https://github.com/lampepfl/dotty/issues/11972):

Here's what happened:

1. Entering "beanproperty" in the search box did not cause anything to happen
2. Entering "bean property" in the search box caused a 404
3. Entering "bean" in the search box dropped me into a random location in the documentation
4. Attempting to find a place to submit an issue ticket was non-trivial

Given how simple these steps were to take, and given how unpredictable the results were, it leaves a terrible first impression.

I don't have the time (and for most others, the motivation) to try and figure these out as it would cause me to lose my current focus (evaluating a project migration LOE [Level Of Effort]) for which I came to this site in the first place.

**Comment by SethTisue** [comment]
4, seconded

**Comment by adpi2** [comment]
Thanks for reporting!

The `BeanProperty` incompatibility described in https://github.com/lampepfl/dotty/issues/11972 is currently missing for the incompatibility table, that is why the search result is irrelevant. I will add the `BeanProperty` incompatibility and make sure the search result is accurate.

Concerning 4, that's really unfortunate, I'll make the link more visible.

**Comment by jim-oflaherty-jr-qalocate-com** [comment]
I just tried out all 4 of my items above. None of them seemed to be resolved.

After reading through the commit, I attempted to also search for "bean-property", and it returns a 404.

**Comment by adpi2** [comment]
The search index was not yet up-to-date. It should be better now.

**Comment by jim-oflaherty-jr-qalocate-com** [comment]
Tysvm! Items 1-3 are now resolved.

I don't see anything regarding item 4, though.

After clicking about a bit, I figured out how to get to "Share Your Feedback" through the "Contribute" link. Given how little it takes for someone to abandon offering correction feedback, is there any way that can be moved somewhere into either the header or footer so it appears at all times? I think the goal would be to take advantage of their impulsive moment following their having an actual issue by encouraging them to immediately submit their issue. Hidden by even one click, the falloff is very large for those who would make the extra effort to share their frustrating issue(s).
